serverless_llm/store/README.md

```diff
@@ -1 +1,37 @@
-# serverless_llm-store
\ No newline at end of file
+# serverless_llm-store
+
+## About
+
+`sllm-store` is an internal library of ServerlessLLM that provides high-performance model loading from local storage into GPU memory. You can also install and use this library in your own projects, following our [quick start guide](https://serverlessllm.github.io/docs/stable/store/quickstart).
+
+## Build From Source
```

Comment:
This is the way to build the wheel. Might be over-complex for local development. I suggest just using `pip install .`

Reply:
Added way to just using `pip install .`